static/js/entry/course_list.js

```diff
@@ -9,10 +9,16 @@ import getMuiTheme from 'material-ui/styles/getMuiTheme';
 import CourseListWithPopover from '../components/CourseListWithPopover';
 
 const courseListEl = document.querySelector('#course-list');
+let courseList = null;
+if (SETTINGS.program) {
+  courseList = SETTINGS.program.courses;
+}
 
-ReactDOM.render(
-  <MuiThemeProvider muiTheme={getMuiTheme()}>
-    <CourseListWithPopover courses={SETTINGS.courses} />
-  </MuiThemeProvider>,
-  courseListEl
-);
+if (courseList && courseListEl !== null) {
```

Comment:
If `SETTINGS.program` exists, `courseList` should always be an array of some length. Can you remove that part of the expression?